src/test/regress/expected/upgrade_list_citus_objects.out

```diff
@@ -273,5 +275,9 @@ ORDER BY 1;
  view citus_worker_stat_activity
  view pg_dist_shard_placement
  view time_partitions
+<<<<<<< HEAD
```

Comment:
Can you please fix the merge conflict issue? Can you also check the merge-to-enterprise test as well?